src/html/404.html

```diff
@@ -0,0 +1,59 @@
+<!doctype html>
+<html class="no-js">
+<head>
+  <meta charset="utf-8">
+  <title>KRIG.io</title>
+
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="description" content="">
+  <meta name="viewport" content="width=device-width,initial-scale=1">
+
+  <!-- COMMON TAGS -->
+  <meta charset="utf-8">
+  <title>KRIG.io</title>
+  <!-- Search Engine -->
+  <meta name="description" content="We are a community of freelancers in tech and digital design.">
+  <meta name="image" content="https://razitazi.com/external/banner.png">
+  <!-- Schema.org for Google -->
+  <meta itemprop="name" content="KRIG.io">
+  <meta itemprop="description" content="We are a community of freelancers in tech and digital design.">
+  <meta itemprop="image" content="https://razitazi.com/external/banner.png">
+  <!-- Twitter -->
+  <meta name="twitter:card" content="summary">
+  <meta name="twitter:title" content="KRIG.io">
+  <meta name="twitter:description" content="We are a community of freelancers within tech and digital design.">
+  <meta name="twitter:site" content="@krig_io">
+  <meta name="twitter:image:src" content="https://razitazi.com/external/banner_twitter.png">
+  <!-- Open Graph general (Facebook, Pinterest & Google+) -->
+  <meta name="og:title" content="KRIG.io">
+  <meta name="og:description" content="We are a community of freelancers in tech and digital design.">
+  <meta name="og:image" content="https://razitazi.com/external/banner.png">
+  <meta name="og:url" content="https://krig.io">
+  <meta name="og:locale" content="sv_SE">
+  <meta name="fb:admins" content="793435117">
+  <meta name="og:type" content="website">
+
+  <link href="//www.google-analytics.com" rel="dns-prefetch">
+  <link href="//ajax.googleapis.com" rel="dns-prefetch">
+  <link href="assets/css/style.css" rel="stylesheet">
+
+  <script src="https://use.typekit.net/qxf5usi.js"></script>
+  <script>try{Typekit.load({ async: true });}catch(e){}</script>
+</head>
+
+<body>
+  <div class="container">
+    <form class="four-oh-four-form">
+      <input type="text" class="404-input">
+    </form>
+
+    <div class="terminal">
```

Comment:
Do we have the styles for the terminal/prompt classes in the style sheets as of now?